src/Service.Tests/DatabaseSchema-MsSql.sql

```diff
@@ -285,6 +286,20 @@ create table books_sold
     last_sold_on_date as last_sold_on,
 )
 
+CREATE TABLE default_with_function_table
+(
+    id INT PRIMARY KEY IDENTITY(1,1),
+    user_value INT,
+    [current_date] DATETIME DEFAULT GETDATE(),
+    current_utc_date DATETIME DEFAULT GETUTCDATE(),
```

Comment:
we should add a test for NOT NULL column, but from GraphQL createItem try omitting the input.